.cbmc-batch/jobs/aws_cryptosdk_edk_list_copy_all/Makefile

```diff
@@ -19,7 +19,7 @@ include ../Makefile.local_default
 
 #########
 # Local vars
-NUM_ELEMS = 2 #Gets full coverage. Property check takes 3m20s on my laptop. 
+NUM_ELEMS = 1 #Pushing just to test. 
```

Comment:
I assume this will need to change back to 2 before merging?

Reply:
Yep, there was a weird CI failure I was trying to debug. Back to 2 and ready to go!